packages/server/src/arpa_reporter/lib/audit-report.js

```diff
@@ -128,13 +128,16 @@ async function getProjectSummaryGroupedByProjectRow(data) {
         ].forEach((columnName) => { row[columnName] = 0; });
     });
 
+    row['Capital Expenditure Amount'] = 0;
+
     // set values in each column
     records.forEach(async (r) => {
         const reportingPeriodEndDate = reportingPeriods.filter((reportingPeriod) => r.upload.reporting_period_id === reportingPeriod.id)[0].end_date;
         row[`${reportingPeriodEndDate} Total Aggregate Expenditures`] += (r.content.Total_Expenditures__c || 0);
         row[`${reportingPeriodEndDate} Total Aggregate Obligations`] += (r.content.Total_Obligations__c || 0);
-        row[`${reportingPeriodEndDate} Total Obligations for Awards Greater or Equal to $50k`] += (record.content.Award_Amount__c || 0);
-        row[`${reportingPeriodEndDate} Total Expenditures for Awards Greater or Equal to $50k`] += (record.content.Expenditure_Amount__c || 0);
+        row[`${reportingPeriodEndDate} Total Obligations for Awards Greater or Equal to $50k`] += (r.content.Award_Amount__c || 0);
```

Comment:
Do you need to initialize these to `0` as above in line 121?

Reply:
@tzinckgraf this is already initialized in L127. :) The reason why the initalization is different for L140 from L135 to L138 is because we're creating these 4 columsn for each recordPeriod, whereas we just need 1 column to encompass the Capital Expenditure Amount among all records.

Reply:
Completely overlooked those lines as they didn't change. Ignore me.